src/Microsoft.Framework.Runtime/NuGet/Xml/ManifestFileXmlExtensions.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Xml.Linq;
+
+namespace NuGet
+{
+    public static class ManifestFileXmlExtensions
+    {
+        public static XElement ToXElement(this ManifestFile manifestFile, XNamespace ns)
+        {
+            var elem = new XElement(ns + "file");
+            elem.SetAttributeValue(ns + "src", manifestFile.Source);
+            elem.SetAttributeValue(ns + "target", manifestFile.Target);
+            elem.SetAttributeValue(ns + "exclude", manifestFile.Exclude);
```

Comment:
Are you sure the attributes should be in the namespace?  You realize that if the namespace is not empty the attribute will have to be prefixed. If the namespace is not bound to a prefix the prefix will be auto generated and you will get this: 
`<file p1:src="file" xmlns:p1="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd" xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd" />`
I think you actually don't want attributes belong to any namespace here - it is quite uncommon in XML to put attributes to a namespace. Typically you would do this only if you want to extend an element that belongs to a non empty namespace and the schema does not allow putting attributes in the empty namespace on this element.